modules/oc-mirror-updating-cluster-manifests-v2.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-updating-cluster-manifests-v2_{context}"]
+= Configuring your cluster to use the resources generated by v2
+
+After you have mirrored your image set to the mirror registry, you must apply the generated ImageDigestMirrorSet (IDMS), ImageTagMirrorSet (ITMS), CatalogSource, and release image signature resources into the cluster.
+
+The ImageDigestMirrorSet resource associates the mirror registry with the source registry and redirects image pull requests using digest specifications. Whereas, the ImagetagMirrorSet redirects the image pull requests using image tags. The CatalogSource resource is used by Operator Lifecycle Manager (OLM) to retrieve information about the available Operators in the mirror registry. The release image signatures are used to verify the mirrored release images.
+
+.Prerequisites
+
+* You have mirrored the image set to the registry mirror in the disconnected environment.
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the OpenShift CLI as a user with the `cluster-admin` role.
+
+. Apply the YAML files from the results directory to the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc mirror --v2 -c 
+clid10.yaml <1>
+file:///home/skhoury/clid-10 <2>
+----
+<1> yaml 
+<2> Insert the file path 
+
+.Example Output for IDMS no-max yaml
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: ImageDigestMirrorSet
+metadata:
+  creationTimestamp: null
+  name: idms-generic-0
+spec:
+  imageDigestMirrors:
+  - mirrors:
+    - localhost:5000/clid10/noflag/openshift-community-operators
+    source: quay.io/openshift-community-operators
+status: {}
+----
+
+.Example Output for IDMS max
```

Comment:
only one idms example is enough, keep only the one above.

Reply:
Done